cores/esp8266/Esp.h

```diff
@@ -168,21 +169,14 @@ class EspClass {
         uint32_t random() const;
 
 #ifndef CORE_MOCK
-        inline uint32_t getCycleCount() __attribute__((always_inline));
+        uint32_t getCycleCount() __attribute__((always_inline)) {
+            return esp_get_cycle_count();
+        }
```

Comment:
I wonder if attribute is required?

Reply:
It is needed. I think it was to force inline when optimizations are turned off. Whatever the case was, if it is not inlined, and this is called e. g. from an ISR, you'll get a crash.